### ISSUE TITLE
Imputer calculates mean along rows not columns 

### ISSUE THREAD

**Issue by QuantumDamage** [EpistasisLab/tpot#565]
https://github.com/rhiever/tpot/blob/1e032375a365d317cf35ffb1eefe431e96b0de6f/tpot/base.py#L740

As far as I understand from [Imputer documentation](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.Imputer.html) we should calculate mean for each column and not each row. Assuming typical Pandas data frame. 

So I guess that line should look more or less like:
 self._fitted_imputer = Imputer(strategy="median", axis=0)

**Comment by weixuanfu** [comment]
Hmm, thank you for finding this bug. We will fix it soon.

**Comment by rhiever** [comment]
Should be fixed on the dev branch.
